docker-compose.yml

```diff
@@ -191,6 +191,17 @@ services:
       - FLINKSTER_CLIENT_ID=${FLINKSTER_CLIENT_ID:?missing/empty}
       - FLINKSTER_SECRET=${FLINKSTER_SECRET:?missing/empty}
     restart: unless-stopped
+    healthcheck:
+      test: ps aux | grep -q 'x2gbfs\.x2gbfs' || exit 1
+      # If you want to wait until all feeds have been generated 
+      # before depending lamassu starts subscribing them, 
+      # configure an appropriate startup_period, e.g. 60s
+      # Otherwise, lamassu might update them only after the 
+      # update interval period has passed, which for test might
+      # last several hours.
+      start_interval: ${X2GBFS_HEALTHCHECK_START_INTERVAL:-1s}
+      start_period: 1s
```

Comment:
AFAICT, having such a low `start_period` doesn't make sense, given that `start_interval` will almost always be higher. 🤔